debian/appscale_install_functions.sh

```diff
@@ -608,8 +608,17 @@ installadminserver()
 
 installhermes()
 {
-    pip install --upgrade --no-deps ${APPSCALE_HOME}/Hermes
-    pip install ${APPSCALE_HOME}/Hermes
+    # Create virtual environment based on Python 3
+    rm -rf /opt/appscale_hermes
+    python3 -m venv /opt/appscale_hermes/
```

Comment:
It appears that the executable for the python3.5 package in trusty is installed at `python3.5` and `python3` points to the python3 package if it is installed.